landoapi/constants.py

```diff
@@ -0,0 +1,2 @@
+# The code freeze dates generally correspond to PST work days.
```

Comment:
This doesn't need its own file, just put it at the top level in the relevant module, `landoapi.models.revisions` seems appropriate.